.github/workflows/build.yml

```diff
@@ -78,6 +78,15 @@ jobs:
           path: offchain/schema.graphql
           if-no-files-found: error
 
+      - name: Install Go
```

Comment:
I left it here to avoid rebuilding the Rust binaries. It's not the prettiest thing but it's efficient. In the future, after we remove the Rust parts the workflow will become a lot simpler.